### DISCUSSION TITLE
"Allow forking" option was not found

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38417] [Repositories] [answered]
Hi.

I've got no "Allow forking" option in ("Settings" section > "Features" panel).

I've read this:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-forking-policy-for-your-repository

Is this documentation obsolete? Or is it something else?

**Comment by airtower-luna** [comment, marked as answer]
Is the repository owned by an organization? The documentation says it's only applicable to private repositories in organizations.

**Reply by ghost** [reply]
> Is the repository owned by an organization?

I've not found this option in any repository owned by me.

> The documentation says it's only applicable to private repositories in organizations.

Okay. So, is a forking allowed for public repositories by default?

**Reply by airtower-luna** [reply]
> So, is a forking allowed for public repositories by default?

Yes. Wouldn't make sense to restrict it: After all, anyone can clone a public repository.

Of course anyone with read-access to a private repository can clone it, too, so I assume it's more about enforcing a desired workflow within the organization.

**Reply by ghost** [reply]
Okay, thank you.
